tests/apps/app_django.py

```diff
@@ -103,23 +109,22 @@ def not_found(request):
 
 
 def complex(request):
-    with opentracing.tracer.start_active_span('asteroid') as pscope:
-        pscope.span.set_tag(ext.COMPONENT, "Python simple example app")
-        pscope.span.set_tag(ext.SPAN_KIND, ext.SPAN_KIND_RPC_SERVER)
-        pscope.span.set_tag(ext.PEER_HOSTNAME, "localhost")
-        pscope.span.set_tag(ext.HTTP_URL, "/python/simple/one")
-        pscope.span.set_tag(ext.HTTP_METHOD, "GET")
-        pscope.span.set_tag(ext.HTTP_STATUS_CODE, 200)
-        pscope.span.log_kv({"foo": "bar"})
+    with tracer.start_as_current_span("asteroid") as pspan:
+        pspan.set_attribute("component", "Python simple example app")
+        pspan.set_attribute("span.kind", "client")
```

Comment:
Let's keep as a pattern the usage of OTel's SpanKind class enumaration. Please adapt the rest of the code to use like this:

```suggestion
from opentelemetry.trace import SpanKind
        pspan.set_attribute("span.kind", SpanKind.CLIENT)
```

Reply:
According to our [instana spec](https://github.ibm.com/instana/technical-documentation/blob/master/tracing/specification/README.md?plain=1#L446), `span.data.sdk.type` must be one of `"entry"`, `"exit"` or `"intermediate"` and in our [codebase](https://github.com/instana/python-sensor/blob/otel_migration/src/instana/span/sdk_span.py#L24), we use `span.kind` to set this attribute.

Reply:
Made the required changes as discussed on the call.